<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>154</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>683</width>
    <height>154</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>683</width>
    <height>154</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label1">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Operator</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioSubtrahieren">
             <property name="text">
              <string>Subtrahieren</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioAddieren">
             <property name="text">
              <string>Addieren</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Zahl 2</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="spinReal2"/>
           </item>
           <item row="1" column="0">
            <widget class="QDoubleSpinBox" name="spinImag2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Zahl 1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="spinReal1"/>
           </item>
           <item row="1" column="0">
            <widget class="QDoubleSpinBox" name="spinImag1"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonBerechnen">
          <property name="text">
           <string>Berechnen</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Ergebnis</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <widget class="QDoubleSpinBox" name="spinImagErgebnis"/>
           </item>
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="spinRealErgebnis"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="labelErgebnis">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelOperator">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="labelCalc">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
